meson/meson-dist-script

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+cd "${MESON_DIST_ROOT}"
+
+# Delete everything we do not want to have in the release tarballs:
+rm -rf \
+   .clang-format \
+   .editorconfig \
+   travis \
+   .github
```

Comment:
The primary use case of meson dist is for running on the developer machine, so you may not actually care about supporting dist on really old versions of Meson.

Meson 0.60.0 supports using `.gitattributes` features like `export-ignore` to delete files you don't want in dist tarballs (and also used by github autogenerated archives).

Reply:
Good to know! I’ll leave this as-is for now for consistency with the i3 meson setup. We can probably adopt this once Meson 0.60 is more common. Even though `ninja dist` is primarily run by me, I don’t want to break it just in case it comes in handy for others.